docs/tutorials/send_receive_ingress.md

```diff
@@ -84,13 +162,27 @@ metadata:
   name: send-receive
   namespace: send-receive-project
 spec:
-  ingressDomain: localresolvconf.com
   acceptors:
     - name: sslacceptor
       port: 62626
-      expose: true
       sslEnabled: true
-      sslSecret: send-receive-sslacceptor-secret
+      expose: true
+      exposeMode: ingress
+      sslSecret: send-receive-sslacceptor-0-svc-ing-ptls
+      ingressHost: send-receive-ss-0.send-receive-hdls-svc.send-receive-project.svc.cluster.local
```

Comment:
The domain `cluster.local` could cause confusion. Can you use a different, i.e. my-domain.com or apps.arkmq.org?

Reply:
I've changed this using a custom dns name in the minikube start options.

Reply:
I still see `cluster.local` in the tutorial. Do you need to push other changes?